lab/admin.py

```diff
@@ -1,10 +1,11 @@
-import reversion
+from reversion.admin import VersionAdmin
+from reversion import revisions as reversion
```

Comment:
@pacharanero do we need this?

Reply:
Probably not, in retrospect. I changed that when I was having an problem with VersionAdmin but it turned out to be caused by a different issue. I will revert, test and push.

Reply:
after Slack discussion we decided to leave it as it is